frontend/src/Dashboard/CrossingListPage/CrossingList.js

```diff
@@ -31,10 +42,12 @@ class CrossingList extends React.Component {
     return (
       <div className='CrossingListContainer'>
         {crossings.map(crossing => 
-          <CrossingListItem
-            crossing={crossing}
-            reasons={statusReasons} 
-            durations={statusDurations} />
+          <div className={classnames({'hiddenCrossing': !crossingIdsToShow.includes(crossing.id)})}>
```

Comment:
After talking about this together, I'm recommend a different approach than having this `div` element here. It's  sole purpose is to potentially attach a class to show or hide itself. I think the component should receive a new prop (ex: `isDisplayed` or `isHidden`) as boolean. Then let the `CrossingListItem` component have full control over how it renders itself. Including the logic whether to re-render or just hide via CSS to preserve values in a dirty form.

If we continue using `{ display: none }` we might need to consider the a11y implications:
http://a11yproject.com/posts/how-to-hide-content/